_src/styles/lazy-styles.css

```diff
@@ -102,6 +102,62 @@
   height: 225px;
 }
 
+.modal-container .modal-content .section.popup-v2:first-child {
```

Comment:
daca functionalitatea asta se leaga cu faptul ca setam in content Template =, atunci putem sa avem un css separat si sa nu-l mai poluam pe asta ( ex: create /template-factories/modal.css )